core/src/main/java/tc/oc/pgm/integration/TranslationIntegrationImpl.java

```diff
@@ -0,0 +1,14 @@
+package tc.oc.pgm.integration;
+
+import java.util.concurrent.CompletableFuture;
+import org.bukkit.entity.Player;
+import tc.oc.pgm.api.integration.TranslationIntegration;
+import tc.oc.pgm.util.translation.Translation;
+
+public class TranslationIntegrationImpl implements TranslationIntegration {
+
+  @Override
+  public CompletableFuture<Translation> translate(Player sender, String message) {
```

Comment:
Is there a reason we need to use `CompletableFuture`s here?

Reply:
if Translations are going to be resolved in an external service they should be handled in a nonblocking way, you can't block the whole server while you wait for an external api to translate a message